utopia-api/src/property-controls/property-controls.ts

```diff
@@ -5,14 +5,12 @@ import { fastForEach } from '../utils'
 interface ControlBaseFields {
   control: RegularControlType
   label?: string
-  defaultValue?: unknown
   visibleByDefault?: boolean
```

Comment:
hmm I thought we have a "placeholder" field too, which makes the defaultValue field redundant, but it doesn't seem to be the case anymore!

is there even a need in the inspector to display a greyed out value for a field which has no value? or we are not showing those controls anyways because we are not rendering those controls because they are collapsed into the "available properties to insert" list below the controls

Reply:
If it has no value, then it should be in that that unused props bit below the controls like you say.